app/components/pool/base/app-packages/app-package-picker.html

```diff
@@ -0,0 +1,4 @@
+<bl-editable-table [formControl]="files">
```

Comment:
files?

Reply:
yeah, not done yet. used the resource file picker as a base.